march_hardware_builder/test/yaml/temperaturesensor/temperature_correct.yaml

```diff
@@ -0,0 +1,2 @@
+slaveIndex: 1
+byteOffset: 2
```

Comment:
Byte offset can be 2000 and the test are still okay with that

Reply:
Same for the slaveIndex

Reply:
Correct, that could happen if your robot becomes very big :)